internal/k8s/ako_init.go

```diff
@@ -277,6 +277,9 @@ func (c *AviController) InitController(informers K8sinformers, registeredInforme
 	var worker *utils.FullSyncThread
 	var tokenWorker *utils.FullSyncThread
 	informersArg := make(map[string]interface{})
+	if lib.GetIpScheme() == lib.IPV6 {
+		utils.AviLog.Infof("AKO is running with IPv6 scheme")
```

Comment:
It would be better to print the scheme in both IPv4 and IPv6 cases since the lib.GetIpScheme() is returning a string.

Reply:
Agreed. Done.